boards/arm/96b_carbon/96b_carbon_defconfig

```diff
@@ -50,3 +50,8 @@ CONFIG_BT_SPI_IRQ_PIN=1
 CONFIG_BT_SPI_RESET_DEV_NAME="GPIOB"
 CONFIG_BT_SPI_RESET_PIN=4
 CONFIG_BT_SPI_MAX_CLK_FREQ=2000000
+
+# BT is the only onboard network iface, so use it for IP networking
+# if it's enabled
+CONFIG_NET_L2_BT=y
+CONFIG_NET_L2_BT_ZEP1656=y
```

Comment:
I am not sure if we should enable the ZEP1656 option by default, this has been discussed in PR #1151